odoo/tools/misc.py

```diff
@@ -1012,7 +1012,7 @@ def clean_context(context):
     """ This function take a dictionary and remove each entry with its key
     starting with ``default_``
     """
-    return {k: v for k, v in context.items() if not k.startswith('default_')}
+    return {k: v for k, v in context.items() if not k.startswith('default_') and not k == 'company_test'}
```

Comment:
mmh not super clean to hack in there for testing purposes. It would be better to extend this method in the tests